lms/djangoapps/ccx/migrations/0003_add_master_course_staff_in_ccx.py

```diff
@@ -80,7 +80,6 @@ class Migration(migrations.Migration):
         ('ccx', '0001_initial'),
         ('ccx', '0002_customcourseforedx_structure_json'),
         ('course_overviews','0010_auto_20160329_2317'), # because we use course overview and are in the same release as that table is modified
-        ('verified_track_content','0001_initial'), # because we use enrollement code and are in the same release as an enrollement related table is created
```

Comment:
Shouldn't we change migrations in a separate PR?

Reply:
I'll mention this in the upstream PR and split into two PRs if needed